### ISSUE TITLE
Aragon?

### ISSUE THREAD

**Issue by yeqbfgxjiq** [aquabu/virtue-dao#1]
This is really cool! Love the idea of using passive income to incentivize and reward a community. Do you guys happen to have a rough idea of how this might work as an Aragon DAO?

**Comment by aquabu** [comment]
Thanks @burrrata - the Aragon DAO can deposit funds into the virtue DAO through a proposal. Basically a virtue DAO allows people to self organize around their common goal from continuous funding or chunks of funding. Happy to chat more about it.

**Comment by yeqbfgxjiq** [comment]
If an Aragon DAO can deposit funds into the VirtueDAO, then really that just means that any address can deposit funds into a VirtueDAO right? In that case, the VirtueDAO is more like a sidecar/app that any Ethereum entity can interact with rather than something that directly integrates. Is that correct?

**Comment by aquabu** [comment]
> If an Aragon DAO can deposit funds into the VirtueDAO, then really that just means that any address can deposit funds into a VirtueDAO right?

Yup, any address can deposit funds into a virtue DAO - e.g. Aragon, DAOStack, Moloch, individual accounts.

>  In that case, the VirtueDAO is more like a sidecar/app that any Ethereum entity can interact with rather than something that directly integrates.

Yes, that's one use case. There could be a deeper integration. The sidecar (nested DAO) approach is the simplest one. 

Some alternate non-sidecar configurations would be:

- Replace the proposal module in an Aragon DAO with a VirtuDAO
- Investor/product purchase/grant $ -> VirtueDAO (no parent DAO)
- rDAI interest -> VirtueDAO (no parent DAO)
